### PR TITLE
Change existing jobs to use new OBMs model.

### PR DESCRIPTION
**Pull request by brianparry**
Depends on: 
https://github.com/RackHD/on-core/pull/164

### CODE REVIEW

**Comment by anhou** [comment]
test this please

**Comment by panpan0000** [comment]
test this please

File: lib/jobs/ipmi-catalog.js [hunk 1]
```diff
@@ -70,35 +58,28 @@ function ipmiCatalogJobFactory(
         var self = this;
         var cmdResp = [];
 
-        waterline.nodes.findByIdentifier(self.nodeId)
-        .then(function (node) {
-            assert.ok(node, 'No node for ipmi catalog');
+        return waterline.obms.findByNode(self.nodeId, 'ipmi-obm-service', true)
+            .then(function (obmSettings) {
+                assert.object(obmSettings, 'No OBM service available.');
+                assert.ok(!_.isEmpty(obmSettings.config), 'No OBM service config available.');
```
Comment (yyscamper) [inline lib/jobs/ipmi-catalog.js]: whether this means the obm will still be pass when inserting an obm without `config`? I think the obm model should prevent those obm service being added.

**Comment by yyscamper** [comment]
:+1:

**Comment by brianparry** [comment]
@panpan0000 @anhou Tests are failing because of the on-core dependency.  We are still working on changes to fix a 1.1 API compatibility issue in on-http.

File: lib/jobs/ipmi-catalog.js (hunk 1 above)
Comment (brianparry) [inline lib/jobs/ipmi-catalog.js]: @yyscamper This is true.  If both the schema validator and the waterline model work properly, we should never have a model without the `config` property.  I added this mainly to make existing unit tests pass.

Comment (yyscamper) [inline lib/jobs/ipmi-catalog.js]: That's OK.

**Comment by srinia6** [comment]
+1

**Comment by benbp** [comment]
👍

**Pull request merged**